### ISSUE TITLE
IllegalStateException when clicking search button in FTS search field

### ISSUE THREAD

**Issue by myatlevEvg** [cuba-platform/cuba#1303]
Platform 7.0

1) Open Refapp 7.0
2) Enable FTS search in jmx console
3) Type something in fts search field
4) Click search button

**AR**
```
java.lang.IllegalStateException: Unknown type of screen ftsSearch
	at com.haulmont.cuba.gui.config.WindowConfig.extractWindowInfoType(WindowConfig.java:121)
	at com.haulmont.cuba.gui.config.WindowConfig$1.getType(WindowConfig.java:89)
	at com.haulmont.cuba.gui.config.WindowInfo.getType(WindowInfo.java:75)
	at com.haulmont.cuba.web.sys.WebScreens.createScreen(WebScreens.java:160)
	at com.haulmont.cuba.web.sys.WebScreens.create(WebScreens.java:156)
	at com.haulmont.cuba.web.sys.WebScreens.openWindow(WebScreens.java:551)
	at com.haulmont.cuba.gui.screen.compatibility.LegacyFrame.openWindow(LegacyFrame.java:100)
	at com.haulmont.cuba.web.gui.components.mainwindow.WebFtsField.openSearchWindow(WebFtsField.java:91)
	at com.haulmont.cuba.web.gui.components.mainwindow.WebFtsField.lambda$new$61446b05$1(WebFtsField.java:75)
```

**Comment by myatlevEvg** [comment]
Check the next issue before close this one: https://app.zenhub.com/workspace/o/cuba-platform/fts/issues/37

**Comment by sukhova** [comment]
Please check the next issue before closing this one: #39

**Comment by jreznot** [comment]
Fixed in #1380
